### PR TITLE
Add some Tencent trackers to spy filter

### PR DESCRIPTION
**Pull request by Hill-98**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

## What issue is being fixed?

`beacon.cdn.qq.com` and `beacon.qq.com` are used for Tencent Beacon, which is Tencent's big data analysis tool.

`aegis-web-sdk` is a front-end monitoring platform launched by Tencent Cloud Monitoring: https://www.npmjs.com/package/aegis-web-sdk

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SpywareFilter/sections/general_url.txt [hunk 1]
```diff
@@ -560,3 +560,4 @@ _analytics.js?
 _pagespeed_beacon?
 _social_tracking.js^
 _tracking.php?b=
+/aegis-sdk/$script
```
Comment (Alex-302) [inline SpywareFilter/sections/general_url.txt:563]: It it regexp type now. Need `*` after the last `/`
```suggestion
/aegis-sdk/*$script
```

File: SpywareFilter/sections/tracking_servers.txt [hunk 2]
```diff
@@ -3001,6 +3005,7 @@
 ||oranges88.com^$third-party
 ||orcapia.com^$third-party
 ||os-data.com^$third-party
+||ourstats.de^$third-party
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers.txt:3008]: We have this rule.

File: SpywareFilter/sections/tracking_servers.txt [hunk 3]
```diff
@@ -76,6 +76,10 @@
 ||us.tags.newscgp.com^
 ||smazaz.icu^
 ||mpraven.org^
+||beacon.cdn.qq.com^
+||beacon.qq.com^
+||aegis.qq.com^
+||rumt-zh.com^
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers.txt:82]: Not sure about rumt-zh.
```suggestion
```

File: SpywareFilter/sections/tracking_servers.txt [hunk 4]
```diff
@@ -76,6 +76,10 @@
 ||us.tags.newscgp.com^
 ||smazaz.icu^
 ||mpraven.org^
+||beacon.cdn.qq.com^
+||beacon.qq.com^
+||aegis.qq.com^
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers.txt:81]: qq => first-party

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/tracking_servers.txt (hunk 2 above)
Comment (Hill-98) [reply SpywareFilter/sections/tracking_servers.txt:3008]: I don't remember adding it, seems like an accident.

File: SpywareFilter/sections/tracking_servers.txt (hunk 3 above)
Comment (Hill-98) [reply SpywareFilter/sections/tracking_servers.txt:82]: It is used by aegis-web-sdk to upload data.

> https://www.npmjs.com/package/aegis-web-sdk

File: SpywareFilter/sections/tracking_servers.txt (hunk 4 above)
Comment (Alex-302) [reply SpywareFilter/sections/tracking_servers.txt:81]: I mean file `tracking_servers_firstparty.txt`

Comment (Hill-98) [reply SpywareFilter/sections/tracking_servers.txt]: it was my oversight.

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
Please never use `push --force`)
image

**Pull request merged**